Report/index.ts

```diff
@@ -18,11 +15,10 @@ import { fileURLToPath } from 'url';
 import childProcess, { ChildProcess, exec, spawn } from 'child_process';
 
 const __dirname = dirname(fileURLToPath(import.meta.url));
-const _root = `${__dirname}/../..`;
+// const _root = `${__dirname}/../..`;
 
-const AZURE_FUNC_TIMEOUT = 2 * 60 * 1000;
+const AZURE_FUNC_TIMEOUT = 1.5 * 60 * 1000;
```

Comment:
We have 2mins in a Azure itself, I usually set a bit less to avoid this hard-stop from the service itself, to have proper error for a broken web apps.